src/sentry/api/endpoints/project_rules.py

```diff
@@ -830,4 +832,13 @@ def post(self, request: Request, project) -> Response:
             wizard_v3=wizard_v3,
         )
 
+        test_event = create_sample_event(
```

Comment:
It seems kinda silly to create a sample event here, but the `RuleProcessor` requires it. I could grab a random latest event from a group in the rule's project instead? Idk if that's any better.